src/http/HttpRequest.ts

```diff
@@ -86,4 +97,10 @@ export class HttpRequest implements types.HttpRequest {
     async text(): Promise<string> {
         return this.#uReq.text();
     }
+
+    clone(): HttpRequest {
+        const newInit = structuredClone(this.#init);

```

Comment:
Question: I see that `structuredClone` was supported starting with Node 17.0.0. What if a customer is on an earlier Node version?